### PR TITLE
DiffEqFlux Benchmarks

### PR DESCRIPTION
**Pull request by avik-pal**
TODOs:
- [x] Neural ODE
- [x] FFJORD (for nested differentiation)
- [x] Neural SDE

### CODE REVIEW

File: src/benchmarkutils.jl [hunk 1]
```diff
@@ -39,3 +39,13 @@ function flatten(results, prefix = "")
...
+
+# Do a forward pass
+function fw(m, ip)
+    CUDA.@sync m(ip)
+end
+
+# Do a forward + backward pass
+function bw(m, ip)
+  gs = CUDA.@sync gradient((m, x) -> sum(m(x)), m, ip)
+end
```
Comment (DhairyaLGandhi) [inline src/benchmarkutils.jl:51]: File endings seem off

**Comment by DhairyaLGandhi** [comment]
Lmk when you think it's good to go.

**Comment by avik-pal** [comment]
it's good to go from my side. We can look into any benchmarking issues later once the CI is setup.

**Comment by DhairyaLGandhi** [comment]
Fair enough, let's get this in!

**Pull request merged**